installation-mac.html

```diff
@@ -84,6 +85,8 @@
 
 <p>You have a choice between versions of Gregorio. To obtain the sources of the stable version, you just need to download the sources from the <a href="https://github.com/gregorio-project/gregorio/releases">download page</a>, and decompress the archive.</p>
 
+<p><b>Nota bene:</b> There are three source code files on the download page.  Two have the name "Source Code" (once in zip and one in tar.gz format).  These files are images of the repository at the time of the release which are added automatically by GitHub.  As a result, the process for building from these files is identical to the process for building from a clone of the repository itself.  The other source code file, "gregorio-x.x.x.tar.bz2", is a special distribution version of the source which has several files, which do not depend on your particular system's architecture, pre-built for you.  As a result, it requires fewer tools to build.</p>
```

Comment:
I'm no wordsmith so I'm a bit at a loss as to *how*, but I think this should somehow stress that you typically want the gregorio-x.x.x.tar.bz2 file.  Maybe mention it first so it's the first thing the reader sees?